content/docs/react-without-jsx.md

```diff
@@ -1,14 +1,14 @@
 ---
 id: react-without-jsx
-title: React Without JSX
+title: React sem JSX
 permalink: docs/react-without-jsx.html
 ---
 
-JSX is not a requirement for using React. Using React without JSX is especially convenient when you don't want to set up compilation in your build environment.
+JSX não é obrigatório para utilizar o React. Utilizar o React sem JSX é especialmente conveniente quando você não quer configurar compilação no seu ambiente de build.
```

Comment:
```suggestion
O JSX não é requisito para utilizar o React. Utilizar o React sem JSX é especialmente conveniente quando você não quer configurar a compilação no seu ambiente de build.
```

Reply:
Hi @fjoshuajr, thank you for the review!
I believe `obrigatório` makes more sense in this case since `requisito` sounds like `requisito de sofware`. `obrigatório` feels like the right meaning as something required for the use of React. What do you think?

Reply:
Please let's discuss it over @glaucia86's comment right below so that I can resolve the conversations. Thank you!